sdk/script/account-manager.ts

```diff
@@ -367,7 +346,7 @@ export class AccountManager {
                     .toString(`base64`)
                     .replace(/\+/g, `_`)  // URL-friendly characters
                     .replace(/\//g, `-`)
-                    .concat(account.id);
+                    .concat(account.email);
```

Comment:
I will address this in a different review - For now, I expect to move the access-key generation onto the server.